.changeset/curly-apples-share.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`Dropdown` - Fixed content being preserved in the DOM when closed
```

Comment:
[suggestion] we should also mention that we're removing the `isOpen` yielded argument
```suggestion
`Dropdown` - Fixed content being preserved in the DOM when closed and removed the `isOpen` yielded argument
```